### PR TITLE
Feature/message build

### PR DESCRIPTION
**Pull request by Qi-Xian**
profile updated.

### CODE REVIEW

File: resources/views/basic/_eduExperience.blade.php [hunk 1]
```diff
@@ -106,7 +106,8 @@
     </div>
 
 </div>
-<div class="form-group" onmouseover="displayExample('eduArea')">
+
+<div class="form-group" onmouseover="displayExample('eduArea1')">
```
Comment (v123582) [inline resources/views/basic/_eduExperience.blade.php]: @Qi-Xian 請問為什麼有 eduArea1 ?

Comment (Qi-Xian) [inline resources/views/basic/_eduExperience.blade.php]: 那個 eduArea1 改成 eduArea 。

---------------------------------------------------------------------
*Best wishes, *
*Qi-Xian Huang*
*Cellphone：0956596370 *
---------------------------------------------------------------------
*◎個人名言：**若要水到渠成，必不辭涓滴細流。*




2015-12-23 8:15 GMT+08:00 WeiYuan <notifications@github.com>:

Comment (v123582) [inline resources/views/basic/_eduExperience.blade.php]: @Qi-Xian 你改完要傳新版的上來啊！！

**Comment by Qi-Xian** [comment]
got it~

---------------------------------------------------------------------
*Best wishes, *
*Qi-Xian Huang*
*Cellphone：0956596370 *
---------------------------------------------------------------------
*◎個人名言：**若要水到渠成，必不辭涓滴細流。*




2015-12-24 20:51 GMT+08:00 WeiYuan <notifications@github.com>:

**Pull request merged**